components/nimbus/ios/Nimbus/FeatureInterface.swift

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import Foundation
+
+/// A small protocol to get the feature variables out of the Nimbus SDK.
+///
+/// This is intended to be standalone to allow for testing the Nimbus FML.
+public protocol FeaturesInterface: AnyObject {
+    /// Get the currently enrolled branch for the given experiment
+    ///
+    /// - Parameter featureId The string feature id that applies to the feature under experiment.
+    /// - Returns A String representing the branch-id or "slug"; or `nil` if not enrolled in this experiment.
+    ///
+    func getExperimentBranch(experimentId: String) -> String?
```

Comment:
I don't think this method needs to be here.

Reply:
Moved into its own protocol to not break iOS and added a note on it that consumer should think about switching to the variables API!